app/controllers/reports/regions_controller.rb

```diff
@@ -34,10 +34,21 @@ def show
     @with_ltfu = with_ltfu?
 
     @children = @region.reportable_children
+
+    repo = Reports::Repository.new(@children, periods: @period, with_exclusions: report_with_exclusions?)
     @children_data = @children.map { |child|
-      Reports::RegionService.new(region: child,
-                                 period: @period,
-                                 with_exclusions: report_with_exclusions?).call
+      slug = child.slug
+      {
+        region: child,
+        controlled_patients: repo.controlled_patients_count[slug][@period],
+        controlled_patients_rate: repo.controlled_patient_rates[slug][@period],
+        uncontrolled_patients: repo.uncontrolled_patients_count[slug][@period],
+        uncontrolled_patients_rate: repo.uncontrolled_patient_rates[slug][@period],
+        missed_visits: repo.missed_visits_count[slug][@period],
+        missed_visits_percentage: repo.missed_visits_rate[slug][@period],
+        registrations: repo.assigned_patients_count[slug][@period],
+        cumulative_patients: repo.cumulative_assigned_patients_count[slug][@period]
+      }
```

Comment:
Does this hash also need other `result` things like `period_info` etc.?

Reply:
No, the children tables don't need it.

I want to move away from the complexity of Result and just use plain hashes like this, so I'm just returning only the key/values we need.